daemon-tests/src/flow.rs

```diff
@@ -7,7 +7,7 @@ use std::time::Duration;
 use tokio::sync::watch;
 
 /// Waiting time for the time on the watch channel before returning error
-const NEXT_WAIT_TIME: Duration = Duration::from_secs(if cfg!(debug_assertions) { 90 } else { 30 });
+const NEXT_WAIT_TIME: Duration = Duration::from_secs(if cfg!(debug_assertions) { 120 } else { 30 });
```

Comment:
🤔 This change doesn't make sense to me based on the commit message.

The number of rollovers in any given test should not affect how long we wait for any individual rollover to complete, which is what `NEXT_WAIT_TIME` controls.

Reply:
Good point! It looks like this change should not have any effect on the test runtime. Could be that I just ran into a longer run while my machine was under heavy load and then prematurely changed this without thinking about it in depth.